docs/software-platforms/general-tools/using-the-terminal.md

```diff
@@ -54,8 +54,29 @@ Note that [Apple provides some additional instructions](https://support.apple.co
 
 Make sure that you have taken all the necessary steps to [install WSL2](../../technical-setup/install-wsl2.md).
 
-The Ubuntu terminal application itself _is_ the terminal.
-Use the Windows Search Bar Menu to search for and open the Ubuntu terminal.
+Accessing the Ubuntu terminal depends on your Windows version:
+
+#### Windows 10
+
+Search for the application called "Ubuntu" in the Windows search menu to launch the Ubuntu terminal.
+
+#### Windows 11
+
+In the most recent Windows 11 version, there is a single application called _Terminal_ which opens all types of shells, aka command line prompts, open in the same window.
```

Comment:
It looks like this is available to Windows 10 users as well: https://learn.microsoft.com/en-us/windows/terminal/ and https://apps.microsoft.com/detail/9n0dx20hk701?rtc=1&hl=en-us&gl=US

We should definitely link to the docs at https://learn.microsoft.com/en-us/windows/terminal/

I wonder if we should just recommend that everyone install it if it is not already installed?

I'm a little worried about the next two lines here as well: do we know that the Ubuntu terminal will always be the default? My guess is that is set somewhere and it could change and/or it defaults to the last thing you used. I would stick with assuming it is not the default and tell people to always click Ubuntu if they want Ubuntu (remove the upper arrow from your screenshot)